locations/spiders/DskBank.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+import scrapy
+from scrapy import Selector
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_BG, OpeningHours
+
+
+class DSKBankBGSpider(scrapy.Spider):
+    name = "dsk_bank_bg"
+    item_attributes = {"brand": "Банка ДСК", "brand_wikidata": "Q5206146"}
+    allowed_domains = ["https://www.dskbank.bg/"]
+    start_urls = ["https://dskbank.bg/контакти/клонова-мрежа/GetOffices/"]
+
+    def parse(self, response):
+        for data in response.json():
+            item = DictParser.parse(data)
+
+            branch_type = data["BranchType"]
+            if "Branch" in branch_type:
+                apply_category(Categories.BANK, item)
+            else:
+                apply_category(Categories.ATM, item)
+
+            if data["Phone"]:
+                item["phone"] = data["Phone"].split(";", 1)[0]
+            else:
+                item["phone"] = None
+
+            item["opening_hours"] = OpeningHours()
+
+            if data["OpenHours"]:
+                item["opening_hours"].add_ranges_from_string(data["OpenHours"], days=DAYS_BG)
+            else:
+                item["opening_hours"] = None
```

Comment:
These two lines can be omitted. It's fine to return a blank `OpeningHours()` object.